t80s_reduce/t80s_sort_night/new.yaml

```diff
@@ -0,0 +1,83 @@
+night: '2016927'
```

Comment:
This configuration file should not be included in the repository. Please, remove it.